worker/options.go

```diff
@@ -29,7 +30,13 @@ func calculateMaxExecsJitter(maxExecs, jitter uint64) uint64 {
 		return 0
 	}
 
-	percent := rand.Intn(int(jitter)) //nolint:gosec
+	random, err := rand.Int(rand.Reader, big.NewInt(int64(jitter)))
+
+	if err != nil {
+		return maxExecs
```

Comment:
Would be worth logging that failure, debug log live.

Reply:
PHP - just get random number from interval
Go - get a degree in math first